1.9/usage/gpu/index.md

```diff
@@ -0,0 +1,21 @@
+---
+post_title: Enable GPU Acceleration
```

Comment:
`nav_title` is optional (for a shorter version of a title which appears in the sidebar). it shouldn't be a duplicate of post_title - you could just leave out in this case.